{"name": "test_backStage_authentication", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "@pytest.mark.run(order=1)\n    @allure.severity(\"blocker\")\n    @allure.description(\"http://10.10.128.152:10052/#/authority/mechanism ÂêéÂè∞Ë¥¶Êà∑ËÆ§ËØÅ\")\n    @allure.testcase(\"http://10.10.128.152:10052/#/authority/mechanism\", \"ÂêéÂè∞Ë¥¶Êà∑ËÆ§ËØÅ üëá\")\n    def test_backStage_authentication():\n        def_name = sys._getframe().f_code.co_name\n        logger.info(\"ÂºÄÂßãÊâßË°åËÑöÊú¨%s:\\n\", def_name)\n    \n        def_name = sys._getframe().f_code.co_name\n        logger.info(\"ÂºÄÂßãÊâßË°åËÑöÊú¨%s:\\n\", def_name)\n    \n        driver = webdriver.Chrome()\n        driver.maximize_window()\n        driver.get(url_back)\n        # Èì∂Ë°åÂØÜÁ†Å‰øÆÊîπ\n>       companyPassForget(driver, company_bank, 1)\n\ntest_flow\\test_Authentication\\test_backStage_authentication.py:80: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntest_flow\\test_Authentication\\test_login.py:122: in companyPassForget\n    waitUntilDisplay(driver, loginOn.txt_pwd_css.value)\ncommon\\BaseFunction.py:13: in waitUntilDisplay\n    WebDriverWait(driver, 30).until(EC.visibility_of_element_located((By.CSS_SELECTOR, ByCss)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"82fa146def491ad13c8276a86e92ab32\")>\nmethod = <selenium.webdriver.support.expected_conditions.visibility_of_element_located object at 0x0000013075ADDD30>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "description": "http://10.10.128.152:10052/#/authority/mechanism ÂêéÂè∞Ë¥¶Êà∑ËÆ§ËØÅ", "attachments": [{"name": "log", "source": "dea9b048-f75e-43cc-90df-5555622cdc4b-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "c153914f-3742-4e7e-a6b5-9970a10df38b-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "dfb91930-baf4-48f8-804d-220ecb4f7c77-attachment.txt", "type": "text/plain"}], "start": 1599114988136, "stop": 1599115032308, "uuid": "6b99b126-7605-4010-86ad-b47ebdf5f9c8", "historyId": "a55d1468f36f1b2cdddeddca0f290d7d", "testCaseId": "e04ecfcfa2b86e3258fad7c48a05b57e", "fullName": "test_flow.test_Authentication.test_backStage_authentication#test_backStage_authentication", "labels": [{"name": "severity", "value": "blocker"}, {"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "test_flow.test_Authentication"}, {"name": "suite", "value": "test_backStage_authentication"}, {"name": "host", "value": "A02-03-00033"}, {"name": "thread", "value": "28600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_flow.test_Authentication.test_backStage_authentication"}], "links": [{"type": "test_case", "url": "http://10.10.128.152:10052/#/authority/mechanism", "name": "ÂêéÂè∞Ë¥¶Êà∑ËÆ§ËØÅ üëá"}]}